packages/admin-ui-docs/docs/teams/contributing/onboarding.mdx

```diff
@@ -51,19 +54,55 @@ Follow the instructions below to get started on development.
 
 To help you get your feet wet and get you familiar with our contribution process, we have a list of good first issues that contain bugs that have a relatively limited scope. This is a great place to get started.
 
-If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment, and assign yourself in the issue, stating that you intend to work on it so other people don’t accidentally duplicate your effort.
+If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment, and assign yourself to the issue, stating that you intend to work on it so other people don’t accidentally duplicate your effort.
 
 ### Sending a Pull Request
 
 The core team is monitoring for pull requests. We will review your pull request and either merge it, request changes to it, or close it with an explanation. We’ll do our best to provide updates and feedback throughout the process.
 
 Before submitting a pull request:, please make sure the following is done:
 
-1.  Clone the repository and create your branch from master.
+1.  Clone the repository and create your branch from main.
 2.  Run yarn: in the repository root.
-3.  If you’ve fixed a bug or added code that should be tested, add tests.
-4.  If you've created a new component or added a new feature, add stories in the storybook.
+3.  If you have fixed a bug or added code that should be tested, add tests.
+4.  If you have created a new component or added a new feature, add stories in the storybook.
 5.  Ensure the test suite passes (yarn test).
 6.  Format your code with prettier (yarn prettier).
 7.  Make sure your code lints (yarn lint).
 8.  Run `yarn build`, yarn `build:storybook` or `yarn build:site` to ensure that the builds are still working.
+
+
+## Contributing with documentation
+Our Admin UI documentation is constantly updated and improved to support you in your journey with the VTEX design system. And to make this a collaborative effort, check out how to contribute with our documentation in the sections below.
+
+### Editing a page
+Our documentation site is built using [Docusaurus 2](https://docusaurus.io/), a static website generator. All of our documentation is stored in the [/docs folder](https://github.com/vtex/admin-ui/tree/main/packages/admin-ui-docs/docs) of our Github Repository.
```

Comment:
```suggestion
Our documentation site is built using [Docusaurus](https://docusaurus.io/), a static website generator. All of our documentation is stored in the [/docs folder](https://github.com/vtex/admin-ui/tree/main/packages/admin-ui-docs/docs) of our Github Repository.
```

I'd remove the version. what matters here is the tool, not its version. Plus if we update in the future to the 3x version, we're covered.